stdlib/Artifacts/test/runtests.jl

```diff
@@ -161,6 +161,5 @@ end
 @testset "`Artifacts.artifact_names` and friends" begin
     n = length(Artifacts.artifact_names)
     @test length(Base.project_names) == n
-    @test length(Base.manifest_names) == n
```

Comment:
Instead of deleting this test, you could just change it to `@test length(Base.manifest_names) == 2n`, right? Similar to how you did below.